device/arista/x86_64-arista_7800r3_48cq2_lc/Arista-7800R3-48CQ2-C48/jr2-a7280cr3-32d4-40x100G.config.bcm

```diff
@@ -860,3 +860,4 @@ dma_desc_aggregator_timeout_usec.BCM8869X=1000
 dma_desc_aggregator_enable_specific_MDB_LPM.BCM8869X=1
 dma_desc_aggregator_enable_specific_MDB_FEC.BCM8869X=1
 sai_pfc_dlr_init_capability=0 
+sai_default_cpu_tx_tc=7
```

Comment:
why we need this property ? this should be controlled from SAI https://github.com/sonic-net/sonic-swss/commit/04f84fc4297d9b3c0e602c76be2cad282fe48fb0

Reply:
@vmittal-msft , can you clarify what is the current behavior without this "default" BCM configuration specification?
Would it be up to the BRCM HWSKUs' decision on setting its own default if this is not specified?  
@abhishekd0 , Do we currently set this as part of the port attribute during port bring up?  If we are not doing it currently, then having a "specified" default setting would be better than don't have it right?  Otherwise, we have no way of knowing which BRCM based HWSKU default setting...

Reply:
This property is needed as part of the BRCM fix for CPU to send LACP/BGP/LLDP packets on queue-7. In the absence of this property and sai fix, cpu will send packets to queue-0. for XGS platforms, we don't need this as cpu tx packets goes to queue-7.

Reply:
i need to check with BRCM if this capability is supported in sai release and can be used instead.

Reply:
@vmittal-msft  - we should request DNX SAI to support this SAI attribute.